_posts/2016-06-22-hello-beercraft.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: Maku brewing and Solita's passion pale ale
+author: Rinorragi
+excerpt: Solita ordered passion pale ale from Maku brewing and got introduction to their craftmenship
+tags: 
+- Beercraft
+---
+Solita wanted to create a beer of their own. The process was much like a software project. First we tried to figure out what we wanted. We had internal voting for the taste and name for the beer and "Intohimo" passion pale ale won the contest. After knowing what we need we went and sent multiple requests for proposals to different breweries. Among the offers we selected Maku brewing and decided with them that the receipt needs to be tested first. Maku brewing created 20l patch of the receipt they invented for us and at 21.6.2016 we got an opportunity to taste the beer and give feedback until final brew. All started with a bus trip from Solita's office at Helsinki to Tuusula. We had a great time already in the bus in discussion of projects and expectations of the upcoming beer. 
+
+## Maku brewing
+
+[Maku brewing](http://www.makubrewing.com/) is 2014 established finnish craft beer brewery making great beers. Their IPA and Amber ale are both award winning beers and really much for my liking. 
+
+After fitting 20 nerds into a bus and travelling to Tuusula, we got an introduction to their brewery by Henri Parviainen who is just leaving the IT business and starting to work full-time at brewery. Their business started when Henri's friend asked Henri and few other guys to invest few hundred euros to 20l Braumeister home brewery system at 2014. Braumeister automates the boiling process for home brewery. After half a year they bought bigger equipments from another brewery and started to make bigger amounts of beer at the same time. Now they have 3000l boiling equipment and 6x 6000l and 4x 2000l fermentation tanks. 
+
+Here we are at the bottling chamber. It serves also as a warehouse. 
+![Tools](/img/hello-beercraft/the_people.jpg)
+
+## Contest for label 
+
+We had our internal contest for best labels. Here is a picture of winning label. 
```

Comment:
We had an internal contest for the label of the beer. Here is a picture of the winning label